cmake/projects/ccv/hunter.cmake

```diff
@@ -25,6 +25,20 @@ hunter_add_version(
      e2b3f7c6093fe94fe746e3825935ae5b4db65f17
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    ccv
+    VERSION
+    "0.7-p5"
+     URL
+     "https://github.com/headupinclouds/ccv/archive/v0.7-p5.tar.gz"
+     SHA1
+     6317031cf45b4eff7900ffcebdb1b940422c240e
+)
+
+
+
+
 hunter_pick_scheme(DEFAULT url_sha1_cmake)
 
 hunter_download(PACKAGE_NAME ccv)
```

Comment:
Wrong repository.  Sorry about that.  I've submitted a new pull request.

On Mon, May 4, 2015 at 11:38 AM, Ruslan Baratov <notifications@github.com>
wrote:

> In cmake/projects/ccv/hunter.cmake
> <https://github.com/ruslo/hunter/pull/134#discussion_r29594573>:
>
> > @@ -25,6 +25,20 @@ hunter_add_version(
> >       e2b3f7c6093fe94fe746e3825935ae5b4db65f17
> >  )
> >
> > +hunter_add_version(
> > +    PACKAGE_NAME
> > +    ccv
> > +    VERSION
> > +    "0.7-p5"
> > +     URL
> > +     "https://github.com/headupinclouds/ccv/archive/v0.7-p5.tar.gz"
> > +     SHA1
> > +     6317031cf45b4eff7900ffcebdb1b940422c240e
> > +)
> > +
> > +
> > +
> > +
> >  hunter_pick_scheme(DEFAULT url_sha1_cmake)
> >
> >  hunter_download(PACKAGE_NAME ccv)
>
> "no newline at end of file" error. Can you please add one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ruslo/hunter/pull/134/files#r29594573>.
>